docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,44 @@
 Version history
 ===============
 
+5.1.9 (2016 April 11)
+---------------------
+
+* Java bug fixes, including:
+   - SDT
+       - fixed width padding calculation for single-pixel image
+   - Deltavision
+       - fixed the parsing of the new date format
+       - added support for parsing and storing the working distance in native units
+   - Micromanager
+       - cleaned up JSON metadata parsing
+   - Olympus Fluoview
+       - fixed null pointer exceptions while parsing metadata
+   - Leica LIF
+       - fixed large multi-tiled files from having incorrect plane offsets after the 2GB mark
+   - EM formats (MRC and Spider)
+       - added native length support for EM readers
+   - Gatan
+       - fixed erroneous metadata parsing
+       - added support for parsing and storing the physical sizes in native units
+   - OME-TIFF
+       - improve handling of OME-TIFF multi-file file-sets with partial metadata blocks
```

Comment:
All the other points are past tense, this should be 'improved' to match. Also 'fileset' is written as one word in the docs